### ISSUE TITLE
KSML tags checklist

### ISSUE THREAD

**Issue by BTCTaras** [BTCTaras/kristscape#16]
### Cursor
- [x] X
- [x] Y
- [x] BR
- [x] RIGHT
- [x] SKIP
- [x] LEFT
- [x] DOWN
- [x] UP
- [x] LF
- [x] CR
- [x] TOP
### Alignment
- [ ] CENTER
- [ ] CELL
- [ ] P
- [ ] W
### Display
- [x] C
- [x] HL
- [x] CLEAR
- [ ] BG
- [ ] HR
- [ ] H
- [x] TITLE
- [x] CLEARTITLE
- [x] CHAR
- [ ] BOX
- [ ] NFP
### Functionality
- [x] END
- [x] SCRIPT
- [ ] SET
- [ ] EAT
- [ ] FEAST
- [ ] A
- [ ] NT
- [ ] D
- [ ] REDIRECT
- [ ] DELINK
- [x] REP
- [x] BLOCK
- [x] NSFW

**Comment by Luca0208** [comment]
Should CLEAR keep the color or remove it?

**Comment by dmarcuse** [comment]
What on earth does the `CLEARTITLE` tag do

**Comment by Luca0208** [comment]
@apemanzilla it probably does this:

[TITLE]My meh title[/TITLE][CLEARTITLE][TITLE]My awesome title[/TITLE]

**Comment by dmarcuse** [comment]
Strange, but it works I guess

**Comment by BTCTaras** [comment]
That's exactly what it does

**Comment by Luca0208** [comment]
When I'm home I'll start with creating some code for a lua sandbox(`script` tag)

**Comment by dmarcuse** [comment]
@Luca0208 I'd be happy to help try escaping from said sandboxes.

**Comment by Luca0208** [comment]
@apemanzilla Ok, gimme a bit of time and I'll push the first sandbox to my fork of the repo

**Comment by Luca0208** [comment]
@apemanzilla the question now is: blacklist or whitelist of functions that you may use in the script?

Edit: If you want I can add you to my repository, also I have no idea what @BTCTaras wants the lua to have

**Comment by dmarcuse** [comment]
Absolutely whitelist. I'd be more than happy to help write the code.

**Comment by Luca0208** [comment]
> Absolutely whitelist. I'd be more than happy to help write the code.

yeah, just thought of it, if you use a blacklist you would need to update it whenever a new CC version comes out. Adding you to the repo now @apemanzilla

**Comment by dmarcuse** [comment]
Alright, we should move further discussion into a new issue about sandboxing and the SCRIPT tag so we don't clutter this up.

**Comment by Luca0208** [comment]
For the center tag please take a look at #25
